src/lib/components/molecules/waffled-squares/index.jsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from "react";
```

Comment:
Please import hooks from `"preact/hooks"`. React should not be referenced anywhere in the project.